### ISSUE TITLE
Implicit circular import in messagetypes package

### ISSUE THREAD

**Issue by g1itch** [Bitmessage/PyBitmessage#1660]
Hello!

I'm trying to build PyBitmessage with pyqtdeploy and just found this: https://github.com/Bitmessage/PyBitmessage/blob/v0.6/src/messagetypes/__init__.py#L6

I cannot imagine how does PyInstaller handle such circular import, it seems impossible ):

```sh
2020-07-09 17:52:30,821 - CRITICAL - Unhandled exception
Traceback (most recent call last):
  File ":/src/bitmessagemain.py", line 54, in <module>
  File ":/src/threads.py", line 19, in <module>
  File ":/src/class_objectProcessor.py", line 17, in <module>
  File ":/src/helper_msgcoding.py", line 8, in <module>
  File ":/src/messagetypes/__init__.py", line 8, in <module>
ImportError: No module named messagetypes
```

**Comment by PeterSurda** [comment]
Yea that seems like a bug. The module has its own test for PyInstaller: https://github.com/Bitmessage/PyBitmessage/blob/v0.6/src/messagetypes/__init__.py#L41

**Comment by g1itch** [comment]
> Yea that seems like a bug. The module has its own test for PyInstaller: https://github.com/Bitmessage/PyBitmessage/blob/v0.6/src/messagetypes/__init__.py#L41

And it's another unnecessary import from parent package which can be replaced with
```python
if getattr(sys, 'frozen', None):
    import message
    import vote
```

**Comment by navjotcis** [comment]
I have tried to reproduce the issue by running Qt app but I was not able to reproduce the issue.

**Comment by navjotcis** [comment]
> > Yea that seems like a bug. The module has its own test for PyInstaller: https://github.com/Bitmessage/PyBitmessage/blob/v0.6/src/messagetypes/__init__.py#L41
> 
> And it's another unnecessary import from parent package which can be replaced with
> 
> ```python
> if getattr(sys, 'frozen', None):
>     import message
>     import vote
> ```

I have removed unnecessary imports.

**Comment by g1itch** [comment]
> I have tried to reproduce the issue by running Qt app but I was not able to reproduce the issue.

Off course you cannot reproduce it by simply running the Qt app. Because this issue makes real problem only for the bundle built by pyqtdeploy. CPython and PyInstaller have no problem with such twisted imports. It's more about how confusing is the code and aim to make it clearer.

**Comment by navjotcis** [comment]
I have built a binary for https://github.com/Bitmessage/PyBitmessage/pull/1674 PR and run it in windows system and try to send a message from the windows machine to the Linux machine and vice versa I have send a message by pressing the shift+send button and everything seems to be work fine and I checked the encoding type in the sent table and it always shows 3 when I try to send a message by pressing shift + send button.

**Comment by PeterSurda** [comment]
fixed in #1674
